### DISCUSSION TITLE
To be able to define the time of conservation of the articles by yourself

### DISCUSSION THREAD

**Discussion by roukmoute** [dailydotdev/daily#539] [Feature Requests]
I have the extension activated when I open a new tab.
I just go to a specific site, so I open a new tab and type the URL.
In the meantime, I still see articles that could interest me, but then they go away by themselves, even though I was only on the empty tab for a few seconds while I typed my URL.
And so I've lost articles several times and I can't find them.

Or maybe there should be a history of the last articles that have been on the feed and are no longer on the front page.

**Comment by idoshamun** [comment]
Hi @roukmoute,

This mechanism is used to ensure you always get "fresh" content.
We understand that it's not the best way to achieve it and we plan to address it during this quarter hopefully.
